### PR TITLE
fixes and utility to add a financial year

### PR DESCRIPTION
**Pull request by jbothma**

### CODE REVIEW

File: budgetportal/models.py [hunk 1]
```diff
@@ -172,6 +172,9 @@ def get_function_budgets(self):
             model_url,
             model_result.elapsed.microseconds / 1000
         )
+        if model_result.status_code == 404:
```
Comment (jbothma) [inline budgetportal/models.py]: Perhaps this should actually be allowed to result in exception and be considered a bug

File: budgetportal/models.py [hunk 2]
```diff
@@ -439,6 +443,8 @@ def get_programme_budgets(self):
...
```
Comment (jbothma) [inline budgetportal/models.py]: Perhaps this should actually be allowed to result in exception and be considered a bug

**Comment by jbothma** [comment]
load_departments tested and working at this stage. Didn't try generating the static site with these though.

File: budgetportal/models.py (hunk 2 above)
Comment (jbothma) [reply budgetportal/models.py]: no this is fine for now - we'll notice very quickly that we're missing this data. And being able to generate the site lets us test things so long

File: budgetportal/management/commands/load_departments.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+from django.core.management.base import BaseCommand
+from budgetportal.models import Department, Government
+import csv
+from django.utils.text import slugify
+import re
+
+
+class Command(BaseCommand):
+    help = 'load programmes'
```
Comment (longhotsummer) [inline budgetportal/management/commands/load_departments.py]: copy paste

**Comment by longhotsummer** [comment]
Cool.

**Pull request merged**